docs/framework/wpf/advanced/walkthrough-hosting-a-3-d-wpf-composite-control-in-windows-forms.md

```diff
@@ -45,7 +45,7 @@ You need the following components to complete this walkthrough:
 <a name="To_Create_the_Windows_Forms_Host_Project"></a>
 ## Create the host project
 
-1. Add a **WPF App (.NET Framework)** project named `WpfUserControlHost` to the solution. For more information, see [Walkthrough: My first WPF desktop application](../getting-started/walkthrough-my-first-wpf-desktop-application.md).
+1. Add a **Windows Forms App (.NET Framework)** project named `WpfUserControlHost` to the solution. For more information, see [Walkthrough: My first WPF desktop application](../getting-started/walkthrough-my-first-wpf-desktop-application.md).
```

Comment:
Removing the link to the WPF tutorial. I cannot find a WinForms equivalent. ack!

```suggestion
1. Add a **Windows Forms App (.NET Framework)** project named `WpfUserControlHost` to the solution.
```